test/auth.test.ts

```diff
@@ -31,7 +32,7 @@ describe('Auth routes tests', () => {
 
       expect(mockSendRegisterEmail).toHaveBeenCalled();
       expect(mockCreate).toHaveBeenCalledWith(
-        { email, name, extra: { lang: DEFAULT_LANG } },
+        { email, name, extra: { lang: DEFAULT_LANG }, password },
```

Comment:
If the create had a password optional, you wouldn't have needed to add the password here.